flink/src/test/java/org/apache/iceberg/flink/TestFlinkSchemaUtil.java

```diff
@@ -54,21 +54,22 @@ public void testConvertFlinkSchemaToIcebergSchema() {
         .field("decimal", DataTypes.DECIMAL(2, 2))
         .field("decimal2", DataTypes.DECIMAL(38, 2))
         .field("decimal3", DataTypes.DECIMAL(10, 1))
+        .field("multiset", DataTypes.MULTISET(DataTypes.STRING().notNull()))
```

Comment:
What happens for a multiset of nullable items?

Reply:
Just like a nullable key in Map, because the default behavior in Flink is nullable key, we support its conversion:
- in the conversion of Flink type to Iceberg type, just ignore the nullable of key.
- in the conversion of Iceberg type to Flink type, the nullable of key becomes false.

Reply:
Okay, for rows that are passed to Iceberg that have null map keys or null values in a multiset, what should happen?

Reply:
Null values are OK, the problem is null keys.
For null keys support, looks like formats are OK, the only constraint of formats is that Avro only support string key of map type.
But the thing is that whether we have any special optimizations for not null. The answer is yes, see `ParquetValueWriters.option`. If a null key comes to parquet writer, I think there should be `NullPointException`. This looks not so elegant.

Another choice is what I said in https://github.com/apache/iceberg/pull/1096/files/8891cd5438306f0b4b226706058beff7c3cd4080#diff-12a375418217cdc6be26c73e02d56065R102 
We can throw a `UnsupportedException` here to tell users, although Flink has default nullable map key.